lib/src/core/bill/bill.dart

```diff
@@ -68,7 +68,7 @@ class Bill {
   bool get isPaymentIdValid {
     var billIdSrt = '$_billId';
     var paymentIdStr = '$_paymentId';
-    if (paymentIdStr.length < 6) return false;
+    if (paymentIdStr.length < 6) false;
```

Comment:
The `return` keyword is needed here. If we remove it from this line, the function do not return false value when length of the `paymentIdStr` value is less than 6 character. So you should add it again.